extensions/elation/actions/createCareGap/createCareGap.ts

```diff
@@ -0,0 +1,47 @@
+import { Category, type Action } from '@awell-health/extensions-core'
+import { makeAPIClient } from '../../client'
+import { type settings } from '../../settings'
+import { dataPoints, fields, FieldsValidationSchema } from './config'
+
+export const createCareGap: Action<
+  typeof fields,
+  typeof settings,
+  keyof typeof dataPoints
+> = {
+  key: 'createCareGap',
+  category: Category.EHR_INTEGRATIONS,
+  title: 'Create care gap',
+  description: 'Create a care gap in Elation.',
+  fields,
+  previewable: true,
```

Comment:
I usually turn this off to make sure previewing care flows doesn't have any side effects in real environments.

Reply:
Good call. Done.